fastlane-plugin-mm_toolkit.gemspec

```diff
@@ -1,35 +1,37 @@
 # coding: utf-8
+# frozen_string_literal: true
 
 lib = File.expand_path("../lib", __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'fastlane/plugin/mm_toolkit/version'
+require "fastlane/plugin/mm_toolkit/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = 'fastlane-plugin-mm_toolkit'
+  spec.name          = "fastlane-plugin-mm_toolkit"
   spec.version       = Fastlane::MmToolkit::VERSION
-  spec.author        = 'Sebastián Varela'
-  spec.email         = 'sebastian.varela@s3ba.net'
+  spec.authors       = ["Sebastián Varela", "Adrián García"]
+  spec.email         = ["sebastian.varela@masmovil.com", "adrian.garcia@masmovil.com"]
```

Comment:
When will I belong to this group of gods?

Reply:
![](https://i.imgur.com/IkdOwkx.jpg)